scripts/data.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3 -i
+# Copyright 2013-2023 The Khronos Group Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+"""Collections of strings and names that affect the projection generation."""
+
+EXCLUDED_EXTENSIONS = (
+    # Atom not projecting right?
+    "XR_MSFT_controller_model",
+    # Projection of static string fails
+    "XR_MSFT_spatial_graph_bridge",
+    "XR_MSFT_spatial_anchor_persistence",
+    "XR_MSFT_holographic_window_attachment",
+    # Projection of UuidMSFT fails
+    "XR_MSFT_scene_understanding",
+    "XR_MSFT_scene_understanding_serialization",
```

Comment:
I'm not actually sure which of these are still broken.  This was something I added back in 1.0.22 when I was patching the code for the previous vcpkg release.  It might be worth re-enabling some or all of them and trying to compile.

Reply:
yeah I agree, I briefly tried enabling a couple of them but didn't look close enough at the results.